activelearning/oracle/oracle.py

```diff
@@ -44,10 +45,11 @@ def __init__(
                 do_domain_map=do_domain_map,
                 device=self.device,
                 float_precision=self.float_precision,
+                negate=negate,
             )
 
         def __call__(self, states):
-            return Branin.__call__(self, states.clone())
+            return Branin.proxy2reward(self, Branin.__call__(self, states.clone()))
```

Comment:
Would this simpler version not do the same?
```suggestion
            return Branin.rewards(states.clone())
```

Reply:
I also think clone is not needed any more, but I'm not sure.

Reply:
this will trigger an endless loop because it calls the __call__ function, which will enter the BraninOracle's call function again

Reply:
Uhm, ok. There must be a way to do this in a simpler way, but it's fine, not a deal breaker.